### PR TITLE
support information_schema.PARAMETERS query in cluster mod

### LINKED ISSUE #24378: Support mainstream database metadata table query
## Background

ShardingSphere has designed its own metadata database to simulate metadata queries that support various databases.

More details:

https://github.com/apache/shardingsphere/issues/21268
https://github.com/apache/shardingsphere/issues/22052

## Task
- [ ] Support query for MySQL metadata `parameters`
- [ ] Support query for MySQL metadata `routines`
- [ ] Support query for MySQL metadata `accounts`

Notice, these issues can be a good example.

https://github.com/apache/shardingsphere/pull/22053
https://github.com/apache/shardingsphere/pull/22057/
https://github.com/apache/shardingsphere/pull/22166/
https://github.com/apache/shardingsphere/pull/22182

### PR DESCRIPTION
**Pull request by lujx98**
Fixes #24378.

Changes proposed in this pull request:
  support information_schema.PARAMETERS query in cluster mod

---

Before committing this PR, I'm sure that I have checked the following options:
- [√] My code follows the [code of conduct](https://shardingsphere.apache.org/community/en/involved/conduct/code/) of this project.
- [√ ] I have self-reviewed the commit code.
- [ ] I have (or in comment I request) added corresponding labels for the pull request.
- [ √] I have passed maven check locally : `./mvnw clean install -B -T1C -Dmaven.javadoc.skip -Dmaven.jacoco.skip -e`.
- [ ] I have made corresponding changes to the documentation.
- [ ] I have added corresponding unit tests for my changes.

### CODE REVIEW

**Comment by strongduanmu** [comment]
Hi @JustRun98, can you take a look for checkstyle error?

**Comment by JustRun98** [comment]
> Hi @JustRun98, can you take a look for checkstyle error?

I have fix it

File: infra/common/src/main/java/org/apache/shardingsphere/infra/metadata/statistics/builder/dialect/MySQLShardingSphereStatisticsBuilder.java [hunk 1]
```diff
@@ -28,38 +28,60 @@
...
+            databaseData.getSchemaData().put(SHARDING_SPHERE, schemaData);
+            result.getDatabaseData().put(SHARDING_SPHERE, databaseData);
         }
-        ShardingSphereSchemaData schemaData = new ShardingSphereSchemaData();
-        for (Entry<String, ShardingSphereTable> entry : shardingSphereSchema.get().getTables().entrySet()) {
-            ShardingSphereTableData tableData = new ShardingSphereTableData(entry.getValue().getName());
-            if (CLUSTER_INFORMATION.equals(entry.getKey())) {
-                tableData.getRows().add(new ShardingSphereRowData(Collections.singletonList(ShardingSphereVersion.VERSION)));
+        
+        Optional<ShardingSphereSchema> informationSchemaSchema = Optional.ofNullable(metaData.getDatabase("information_schema")).map(database -> database.getSchema("information_schema"));
```
Comment (tuichenchuxin) [inline infra/common/src/main/java/org/apache/shardingsphere/infra/metadata/statistics/builder/dialect/MySQLShardingSphereStatisticsBuilder.java:70]: I think `information_schema` should in `shardingsphere` database.

**Review by tuichenchuxin — CHANGES_REQUESTED** [review]

File: infra/common/src/main/java/org/apache/shardingsphere/infra/metadata/statistics/builder/dialect/MySQLShardingSphereStatisticsBuilder.java (hunk 1 above)
Comment (JustRun98) [reply infra/common/src/main/java/org/apache/shardingsphere/infra/metadata/statistics/builder/dialect/MySQLShardingSphereStatisticsBuilder.java:70]: @tuichenchuxin
are you guys expect that all the metadata table's data should presist in shardingsphere database?
I think if so the databases like information_schema will be totally useless

Comment (tuichenchuxin) [reply infra/common/src/main/java/org/apache/shardingsphere/infra/metadata/statistics/builder/dialect/MySQLShardingSphereStatisticsBuilder.java:70]: it'll be like this.
```
database:shardingsphere
- schema: information_schema
```

**Comment by strongduanmu** [comment]
Since this pr has long time no response, I will close it. If you are interested, you are welcome to submit a new PR.

**Pull request closed without merging**